api/src/RestServer.php

```diff
@@ -147,6 +147,12 @@ static function mapHgResponse($hgrCode) {
 				$httpcode = "202 Accepted";
 				$codeString = 'INPROGRESS';
 				break;
+			case HgResumeResponse::TIMEOUT:
+				// 408 is not currently handled by the client, so it will be
```

Comment:
we should specify the client version when we say it's not supported, and then update the comment when we know what client version does support it.

Reply:
This is for the current version. If we want to make the client keep polling we basically have to send it something it's not expecting, so I just tried to pick a status code that sort of made sense. It's suboptimal, because the client will retry the chunk and we'll just ignore it, but as far as I can tell there isn't a better solution for the current Chorus clients.

We could check if the client is accessing v4 and then return 202 + "INPROGRESS" instead. The client would have to be adapted for that of course.

But, this is for v3 right now.

Reply:
This is a good enough solution and should hopefully fix v3.

Future work is to publish v4 (or maybe it's v5?) and fix some things in the chorus client as well that use only v5.